cognite_toolkit/cognite_modules/examples/cdf_data_pipeline_3d_valhall/data_sets/dataset.yaml

```diff
@@ -0,0 +1,14 @@
+externalId: ds_3d_{{location_name}}
```

Comment:
This is duplicated of https://github.com/cognitedata/toolkit/blob/main/cognite_toolkit/cognite_modules/examples/cdf_oid_example_data/data_sets/data_sets.yaml#L10

Should remove if from the OID example?

Reply:
This data set contains metadata about the source and used raw tables, so don't think we should remove.